landing page/css/main.css

```diff
@@ -0,0 +1,76 @@
+@font-face {
+	font-family: hLight;
```

Comment:
Set the default tab spaces to 2. The code looks clean from [L58-76](https://github.com/gawdsnitkkr/tedxnitkurukshetra-frontend/blob/d88c8abe3e21b17396d7002f55e16f254a09fe92/landing%20page/css/main.css#L58-L76). Fix the indentation from line from line 1 to line 58.

Follow the same coding convention before in all the files.